restcomm/restcomm.http/src/main/java/org/restcomm/connect/http/AccountsEndpoint.java

```diff
@@ -506,12 +510,22 @@ private Account prepareAccountForUpdate(final Account account, final Multivalued
                 } else
                     throw new AuthorizationException();
             }
-        } catch (AuthorizationException | AccountAlreadyClosed | PasswordTooWeak e) {
+            if (data.containsKey("EmailAddress")) {
+                String newEmailAddress = data.getFirst("EmailAddress").toLowerCase();
```

Comment:
any reason to force lowerCase here? AFAIK the user/local part is case sensitive form SMTP spec...

Reply:
In `createForm` function I see the created email is always lower case so that is the reason I forced lower case here to be consistent with creating process.